x-pack/plugins/security_solution/public/management/pages/event_filter/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExceptionListType } from '../../../../../lists/common/shared_exports';
+
+export const EVENT_FILTER_LIST_ID = '.endpointEventFilterList';
```

Comment:
To bee agreed upon

Reply:
The list id, name and description should be retrieved (`import`) from here https://github.com/elastic/kibana/blob/9a0c73e515f0e0c7ff88d6b06df46aa34cd8d84f/x-pack/plugins/lists/common/constants.ts#L64-L71 

They are already being used in the Exceptions server service as well - https://github.com/elastic/kibana/blob/9a0c73e515f0e0c7ff88d6b06df46aa34cd8d84f/x-pack/plugins/lists/server/services/exception_lists/create_endoint_event_filters_list.ts#L40-L64